### PR TITLE
 integrate limit request rate extension by cfn template

### PR DESCRIPTION
**Pull request by heqiqi**
*Issue #, if available:*
new feature, not issue
*Description of changes:*
limit the request rate lower than 100query/5min by lamabda@edge
*How Has This Been Tested:*
yes. I deploy this cloudformation and tested it .
*[x] My testing has passed*

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: docs-ig/en/deployment.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ Click below deploy button to deploy this solution in your AWS account. As for La
...
 | [Verify JSON web token](https://github.com/awslabs/aws-cloudfront-extensions/tree/main/function/js/verify-jwt) |  [![Deploy](../images/deploy_button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=verify-jwt&templateURL=https:%2F%2Faws-cloudfront-extensions-cff.s3.amazonaws.com%2Fverify-jwt.yaml)                                       |
 | [Customize request host](https://github.com/awslabs/aws-cloudfront-extensions/tree/main/function/js/custom-host) | [![Deploy](../images/deploy_button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=custom-host&templateURL=https:%2F%2Faws-cloudfront-extensions-cff.s3.amazonaws.com%2Fcustom-host.yaml)                                     |
+| [Limit Request Rate](https://github.com/awslabs/aws-cloudfront-extensions/tree/main/function/js/limit-request-rate) | [![Deploy](../images/deploy_button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=limit-request-rate&templateURL=https:%2F%2Faws-cloudfront-extensions-cff.s3.amazonaws.com%2Flimit-request-rate.yaml)  |
```
Comment (NingLu) [inline docs-ig/en/deployment.md:96]: Remove this line as the cfn template link is not publicly available, we will add it in implementation guide once the testing is completed

File: docs-ig/zh/deployment.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ CloudFormation堆栈部署了如下模块：
...
 | [Verify JSON web token](https://github.com/awslabs/aws-cloudfront-extensions/tree/main/function/js/verify-jwt) |  [![Deploy](../images/deploy_button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=verify-jwt&templateURL=https:%2F%2Faws-cloudfront-extensions-cff.s3.amazonaws.com%2Fverify-jwt.yaml)                                       |
 | [Customize request host](https://github.com/awslabs/aws-cloudfront-extensions/tree/main/function/js/custom-host) | [![Deploy](../images/deploy_button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=custom-host&templateURL=https:%2F%2Faws-cloudfront-extensions-cff.s3.amazonaws.com%2Fcustom-host.yaml)                                     |
-
+| [Request速率限制](https://github.com/awslabs/aws-cloudfront-extensions/tree/main/function/js/limit-request-rate) | [![Deploy](../images/deploy_button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=limit-request-rate&templateURL=https:%2F%2Faws-cloudfront-extensions-cff.s3.amazonaws.com%2Flimit-request-rate.yaml)
```
Comment (NingLu) [inline docs-ig/zh/deployment.md:99]: Remove this line as the cfn template link is not publicly available, we will add it in implementation guide once the testing is completed

Comment (heqiqi) [reply docs-ig/zh/deployment.md]: already remove description in deployment.md

**Review by NingLu — APPROVED** [review]
Looks good, ship it

**Pull request merged**